x-pack/plugins/maps/public/connected_components/toolbar_overlay/toolbar_overlay.test.tsx

```diff
@@ -33,10 +35,12 @@ test('Should only show set view control', async () => {
 test('Should show all controls', async () => {
   const component = shallow(
     <ToolbarOverlay
+      showToolsControl={false}
```

Comment:
Why did showToolsControl change from true to false? Now this test no longer "Should show all controls". Change showToolsControl back to true.